### ISSUE TITLE
disableMinute not obeyed using DateTimeInput

### ISSUE THREAD

**Issue by mweiczorek** [arfedulov/semantic-ui-calendar-react#158]
When using `disableMinute` in the `DateTimeInput`, the minute view still appears.

Version 0.15.0

```
<DateTimeInput
  dateFormat={Temporal.dateTimePickFormat}
  value={bookingStore.selectedDateTime}
  timeFormat="AMPM"
  closable
  disableMinute
  preserveViewMode={false}
  onChange={BookingFormHandler.onDateTimeSelected} />
```

**Comment by alshakero** [comment]
Hi! Would you accept a PR for this @arfedulov?

**Comment by arfedulov** [comment]
fixed #165
